server/schemas/typeDefs.js

```diff
@@ -34,6 +34,22 @@ const typeDefs = `
 
   type Query {
     me: User
+    bookSearch(query: String!): [Book]
+    getBooks: [Book]
+    getSingleBook(bookId: ID!): Book
+    orderHistory(userId: ID!): [Order]
+  }
+
+  type Mutation {
+    addUser(username: String!, email: String!, password: String!): Auth
+    signIn(username: String!, password: String!): Auth
+    changeUsername(userId: ID!, username: String!): Auth
+    changeEmail(userId: ID!, email: String!): Auth
+    changePassword(userId: ID!, password: String!): Boolean
+    signOut(userId: ID!): Boolean
+    addToCart(userId: ID!, bookISBN: String!): Book
+    saveBook(userId: ID!, bookISBN: String!): Boolean
+    removeItemFromCart(userId: ID!, bookISBN: String!): Book
```

Comment:
What Book are we returning here? If it is being removed from the cart then we should really only be caring about whether that was successful or not.

Reply:
i was thinking we could send a message to the user telling them they removed that book from the cart. We could take it out tho